public/js/pages/localweb.js

```diff
@@ -20,9 +20,35 @@ requirejs.config({
   }
 });
 
-require(["jquery", "languages", "selectize", "transition", "collapse", "carousel", "magnific-popup"], function ($, languages, selectize ) {
+require(["jquery", "analytics", "languages", "selectize", "transition", "collapse", "carousel", "magnific-popup"], function ($, analytics, languages, selectize ) {
   "use strict";
   var $window = $(window);
+  var TIME_DELAYED_REDIRECT = 1000; // in milliseconds
+  // analytics
+  var delayRedirect = function(url) {
+    setTimeout(function() {
+      window.location.href = url;
+    }, TIME_DELAYED_REDIRECT);
+  };
+  $("#header-partner-with-us-link").click(function() {
```

Comment:
this element now links to a same page anchor, not sure if we want to record this?